cli/cmd/compliance_gcp.go

```diff
@@ -369,10 +370,9 @@ Supported report types are: CIS_1_0, CIS_1_2
 To show the current status of recommendations in a report run:
 	lacework compliance gcp status CIS_1_2
 
-To enable all recommendations for CIS_1_1 report run:
+To enable all recommendations for CIS_1_2 report run:
 	lacework compliance gcp enable CIS_1_2
```

Comment:
are you planning to update all other compliance commands? or wait until we "again" review and unhide them?

I would appreciate if we update all of them but it is your call, as long as you remember to do so.

Reply:
I had not planned on changes to other compliance commands in this PR. It would make sense to allow the CIS version to be an accepted value.

Reply:
Created a ticket to tackle this at a later date -> https://lacework.atlassian.net/browse/ALLY-982